parts/k8s/kubernetesagentcustomdata.yml

```diff
@@ -15,6 +15,13 @@ write_files:
   content: !!binary |
     {{WrapAsVariable "sshdConfig"}}
 
+- path: "/etc/systemd/system.conf"
```

Comment:
This needs to be added to the VMSS custom data as well

Reply:
my bad that's only for master custom data (there is no agent custom data for vmss)